build/Makefile

```diff
@@ -14,6 +14,13 @@ ifndef ENABLE_DEBUG
 $(error "ENABLE_DEBUG is not set.  Please re-run configure")
 endif
 
+# Include the version file
+include ../../docker.version
+
+ifndef CONTAINER_BUILDVERSION_STATUS
+$(error "Is Makefile.version missing?  Please re-run configure")
```

Comment:
If `CONTAINER_BUILDVERSION_STATUS` is defined in `docker.version` then the error message should be fixed.